### PR TITLE
fix: `hs projects logs` links and reduce number of prompts

### PR DESCRIPTION
**Pull request by joe-yeager**
## Description and Context

- Depends on https://github.com/HubSpot/hubspot-local-dev-lib/pull/181

Updates made `hs project logs`
- Fix links to the UI to be deep links
- Reduces number of prompts to 1 single select that asks which function to check for.  

## Screenshots

https://github.com/user-attachments/assets/e5951719-e5ca-4d78-a8d5-5b3e29fa6fa4

## TODO

- [ ] Publish LDL once https://github.com/HubSpot/hubspot-local-dev-lib/pull/181 is merged and bump the deps

## Who to Notify

@camden11 @kemmerle @brandenrodgers

### CODE REVIEW

File: packages/cli/commands/project/logs.js [hunk 1]
```diff
@@ -4,253 +4,154 @@ const {
...
-//             {
-//               after,
-//             }
-//           )
-//         : getFunctionLogs(accountId, functionName, { after });
-//     } catch (e) {
-//       handleLogsError(e, functionName, projectName);
-//     }
-//   };
+  // const { latest, follow, compact, functionName } = options;
```
Comment (brandenrodgers) [inline packages/cli/commands/project/logs.js:37]: If you want, you can totally just delete all this commented code. We have the commit history if we ever want to go back and reference it

File: packages/cli/commands/project/logs.js [hunk 2]
```diff
@@ -4,253 +4,154 @@ const {
...
-//     latest,
-//     follow,
-//     compact,
-//     appPath,
-//     functionName,
-//     projectName,
-//   } = options;
-
-//   let logsResp;
+const handleFunctionLog = async (accountId, options) => {
```
Comment (kemmerle) [inline packages/cli/commands/project/logs.js:34]: Clarification: Does this function only return the log "No logs found"? Assuming this is temporary until the BE catches up.

Comment (joe-yeager) [reply packages/cli/commands/project/logs.js:34]: Yep, I'll add a clarifying comment

File: packages/cli/lib/projectLogsManager.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
+const { getProjectConfig, ensureProjectExists } = require('./projects');
+const {
+  fetchProjectComponentsMetadata,
+} = require('../../../../hubspot-local-dev-lib/dist/api/projects');
```
Comment (kemmerle) [inline packages/cli/lib/projectLogsManager.js:4]: Leaving a note here to change this before merging.

File: packages/cli/commands/project/logs.js [hunk 4]
```diff
@@ -4,253 +4,154 @@ const {
...
+  //   logsResp = await ProjectLogsManager.fetchLatest();
+  // } else {
+  //   logsResp = await ProjectLogsManager.tailCall();
+  // }
+  //
+  // if (logsResp) {
+  //   outputLogs(logsResp, options);
+  // }
+  // return !!logsResp;
+  logger.log(i18n(`${i18nKey}.logs.noLogsFound`, { name: functionName }));
```
Comment (brandenrodgers) [inline packages/cli/commands/project/logs.js:58]: Might make sense to mute this no logs found message for now.

File: packages/cli/lib/projectLogsManager.js (hunk 3 above)
Comment (joe-yeager) [reply packages/cli/lib/projectLogsManager.js:4]: Whoops, great catch.  My IDE auto import went wild with this one.

**Review by kemmerle — APPROVED** [review]
Had some minor comments, but nothing that isn't easy to address. Love that we've simplified the command so much, I remember how confusing it was 🥳

File: packages/cli/commands/project/logs.js (hunk 4 above)
Comment (joe-yeager) [reply packages/cli/commands/project/logs.js:58]: Sounds good, I'll delete it for now

File: packages/cli/lib/projectLogsManager.js (hunk 3 above)
Comment (joe-yeager) [reply packages/cli/lib/projectLogsManager.js]: https://github.com/HubSpot/hubspot-cli/pull/1131/commits/f88732c070b9f991f8fbdd5ee8094ec353328126

File: packages/cli/commands/project/logs.js (hunk 4 above)
Comment (joe-yeager) [reply packages/cli/commands/project/logs.js]: https://github.com/HubSpot/hubspot-cli/pull/1131/commits/f88732c070b9f991f8fbdd5ee8094ec353328126

File: packages/cli/commands/project/logs.js (hunk 2 above)
Comment (joe-yeager) [reply packages/cli/commands/project/logs.js]: I actually ended up deleting it instead https://github.com/HubSpot/hubspot-cli/pull/1131#discussion_r1735044498

File: packages/cli/commands/project/logs.js (hunk 1 above)
Comment (joe-yeager) [reply packages/cli/commands/project/logs.js]: https://github.com/HubSpot/hubspot-cli/pull/1131/commits/f88732c070b9f991f8fbdd5ee8094ec353328126

File: packages/cli/lang/en.lyaml [hunk 5]
```diff
@@ -586,16 +586,18 @@ en:
         logs:
           describe: "Get execution logs for a serverless function within a project"
           errors:
-            invalidAppName: "Could not find app with name \"{{ appName }}\" in project \"{{ projectName }}\""
+            noProjectConfig: "No project detected. Please run this command again from a project directory."
+            failedToFetchProjectDetails: "Failed to fetch project details"
+            unableToDetermineProject: "Unable to determine which project to use"
+            unableToSetFunction: "Unable to set function, no functions to choose from"
+            noFunctionWithName: "No function with name \"{{ name }}\""
```
[the remark this answers is not in the document]
Comment (joe-yeager) [reply packages/cli/lang/en.lyaml:598]: https://github.com/HubSpot/hubspot-cli/pull/1131/commits/060b4be63e9bac6974f9a4222f14a4410733081d

**Review by brandenrodgers — APPROVED** [review]

**Pull request merged**